src/components/Global/BaseSnackbar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-snackbar
+    v-model="showSnackBar"
+    :color="color"
+    class="base-snackbar"
+    position="absolute"
```

Comment:
This fixes the position issue combined with the location prop. If we need more props here to be proxied, we'll have to add them manually to this component. As-is this cleans up a lot though.